### PR TITLE
Shredders

### PR DESCRIPTION
**Pull request by IData-RedactedI**
## About the PR

Added peaceable but non functional shredders
Hopefully this is enough functionality that others can add to 

## Why / Balance
Its on "[THE LIST](https://github.com/new-frontiers-14/frontier-station-14/issues/166)"

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

### CODE REVIEW

**Comment by IData-RedactedI** [comment]
Hey Checkraze, this is unfortunately as far as I can go when it comes to the shredders. I think whatever systems you want to add can be grafted onto the framework here though.

**Comment by Cheackraze** [comment]
Sure thing! I won't have a chance to review for a day or two, but thank you very much for your contribution!

File: Resources/Prototypes/_NF/Entities/Structures/Machines/shredder.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+- type: entity
+  id: Shredder
+  parent: BaseLathe
+  name: shredder
+  description: It shreds things. What more is there to say?
+  components:
+  - type: Sprite
+    sprite: _NF\Structures\Machines\Shredder.rsi
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Entities/Structures/Machines/shredder.yml:8]: while this likely still works in game, it makes the yaml linter cry, so I'll switch these to forward slashes instead

File: Resources/Textures/_NF/Structures/Machines/syndicate_shredder.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          [
+            0.1,
+            0.1,
+            0.1,
+            0.1,
+            0.1,
+            0.1
+          ]
+        ]
+      }
```
Comment (Cheackraze) [inline Resources/Textures/_NF/Structures/Machines/syndicate_shredder.rsi/meta.json:37]: just missing a comma here it appears, i'll go ahead and fix it

**Review by Cheackraze — APPROVED** [review]

**Comment by Cheackraze** [comment]
excellent. The current functionality is whatever, since this is planned to be a part of a much bigger system in the near future.

**Pull request merged**